src/main/webapp/js/captions.js

```diff
@@ -229,6 +235,17 @@ function sendJsonForm(json) {
         });
 }
 
+function resizeIFrame() {
```

Comment:
Add method header and comment the method (I think I wrote this function, so I'll do it)